### ISSUE TITLE
Items added with the script are not exported via caldav

### ISSUE THREAD

**Issue by piggz** [buzz/ics2owncloud.py#18]
Hi, i dont know if the problem is with the script, or the dav functionality, but items added by the script, which appear in the calendar gui, are not shared via the caldav endpoint, when items manually added to the same calendar are.

**Comment by mossroy** [comment]
Strange, I see all events from Thunderbird (as well as from the gui)

**Comment by piggz** [comment]
I only see manually added events in both the sailfish calendar, and korganiser in kde.  The online calendar view shows all my imported events.

**Comment by mossroy** [comment]
Works fine for me on Nextcloud 10.0 + Calendar 1.4.0, both from Firefox OS 2.5 and Thunderbird+Lightning.
Did you double-check you don't upload the ics to another nextcloud/owncloud calendar, that you did not subscribe to on sailfish & kde?

**Comment by piggz** [comment]
Yes, i checked im subscribed to the correct calendar in sailfish/korganiser.... the calendar is called 'work' and shows all the events in the web view, when new events are added from the calendar app, they sync to the phone/korganiser, but the ones from ics2owncloud do not show ... i am also using the 1.4 app, a fresh calendar, and a fresh run of ics2owncloud.  KOrganiser correctly shows the new calendar, but nothing in it.  I also tried the new calendar subscription feature, but subscribed calendars are not shared over caldav at all, so this script is still useful for the time being.
